synapse/tools/cellauth.py

```diff
@@ -235,6 +253,8 @@ def makeargparser():
     muxp.add_argument('--addrule', help='Add the given rule to the user/role.')
     muxp.add_argument('--delrule', type=int, help='Delete the given rule number from the user/role.')
 
+    pars_mod.add_argument('--object', type=str, help='The object to apply new rule to in kind:iden format')
```

Comment:
`The object to apply new rule to in kind:iden format`

shouldn't this just be iden? i don't see type used in the test.

Reply:
good catch